boards/nxp/imx91_evk/Kconfig.imx91_evk

```diff
@@ -0,0 +1,6 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_IMX91_EVK
+	select SOC_MIMX9131 if BOARD_IMX91_EVK_MIMX9131
```

Comment:
is this board going to have other SoCs adding to it? If not, then
```suggestion
	select SOC_MIMX9131
```

Reply:
No other SoC, will take your suggestion. Thanks